app/uk/gov/hmrc/helptosave/connectors/EligibilityCheckConnector.scala

```diff
@@ -43,12 +43,16 @@ class EligibilityCheckConnectorImpl @Inject() (http: WSHttp, metrics: Metrics) e
   def url(nino: String): String =
     s"$itmpBaseURL/help-to-save/eligibility-check/$nino"
 
+  val headers: Map[String, String] = Map(
+    "Environment" → getString("microservice.services.itmp-eligibility-check.environment")
```

Comment:
strange way of configuring the host config , but if its their style of doing it , we are ok